mods/d2k/rules/vehicles.yaml

```diff
@@ -649,7 +649,7 @@ combat_tank_o:
 	Turreted:
 		TurnSpeed: 20
 	Encyclopedia:
-		Description: The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.\n\nThe Ordos Combat Tank is faster but weaker.
+		Description: The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.\n\nThe Ordos Combat Tank is faster than the Atreides Combat Tank, but weaker.
```

Comment:
This reads weird to me, Harkonnen one is slower than Atreides one, so naturally Ordos one is the fastest. But this sounds like it is only faster than Atreides one. `than other Combat Tanks`, or `than its counterparts` would better imo.